app/controllers/movies_controller.rb

```diff
@@ -38,8 +48,15 @@ def top_40
     json1 = JSON.parse(page1.body, symbolize_names: true)
     json2 = JSON.parse(page2.body, symbolize_names: true)
 
-    movies = json1[:results] + json2[:results]
+    make_movies(json1, json2)
+
+    @movies_info = Movie.all
+  end
 
-    @movies_info = movies.pluck(:title).zip(movies.pluck(:vote_average))
+  def make_movies(json1, json2)
+    movies = json1[:results] + json2[:results]
+    movies.each do |movie|
+      Movie.create(title: movie[:title], vote_average: movie[:vote_average], movie_db_id: movie[:id])
```

Comment:
I really dont know if I agree with storing the movie data. One of the problems with this is that it stores that info on our server and stays there, meaning that every time this page is used our database fills up more and more.

Reply:
I'm glad we talked about this!  I've taken it out and now we're just doing an API call for it!